nci_seer/web_client/templates/itemView.pug

```diff
@@ -8,7 +8,7 @@
     finished: ['quarantine']
   };
   var buttons = {
-    process: {name: 'Process', title: 'Keep a copy of the original image, redact marked fields, and move the result to the Audited location'},
+    process: {name: 'Process', title: 'Keep a copy of the original image, redact marked fields, and move the result to the processed folder'},
```

Comment:
sorry to pick nits, could you use consistent capitalization in these titles? currently only the `finish` has the name of the target location capitalized and the others do not, i.e. `'Move this image to the Finished location'`

Reply:
Will do.  I appreciate the feedback.